### ISSUE TITLE
Missing textarea index.js under components dir on npm-installed package

### ISSUE THREAD

**Issue by litneet64** [kor-ui/kor#53]
Hey! Working on a project with Kor I realized there's no `index.js` under `@kor-ui/kor/components/textarea` when you install it from `npm` on latest version (1.5.0), but strangely enough it's included inside the bundled js (`@kor-ui/kor`), which means you can't really import nor use the textarea comp as a **single** component but rather only if you import the bundled js.

Demo:
```bash
root@6316c9055ae6:~# npm install @kor-ui/kor@1.5.0

added 3 packages, and audited 4 packages in 4s

found 0 vulnerabilities
npm notice 
npm notice New minor version of npm available! 7.3.0 -> 7.4.0
npm notice Changelog: https://github.com/npm/cli/releases/tag/v7.4.0
npm notice Run npm install -g npm@7.4.0 to update!
npm notice 
root@6316c9055ae6:~# ls -lah node_modules/@kor-ui/kor/components/textarea
total 16K
drwxr-xr-x  2 root root 4.0K Jan 10 04:43 .
drwxr-xr-x 38 root root 4.0K Jan 10 04:43 ..
-rw-r--r--  1 root root   46 Oct 26  1985 index.d.ts
-rw-r--r--  1 root root 1.2K Oct 26  1985 kor-textarea.d.ts
root@6316c9055ae6:~# ls -lah node_modules/@kor-ui/kor/components/text    
total 44K
drwxr-xr-x  2 root root 4.0K Jan 10 04:43 .
drwxr-xr-x 38 root root 4.0K Jan 10 04:43 ..
-rw-r--r--  1 root root   38 Oct 26  1985 index.d.ts
-rw-r--r--  1 root root  25K Oct 26  1985 index.js
-rw-r--r--  1 root root  898 Oct 26  1985 kor-text.d.ts
```
Error while resolving import paths with webpack inside my class `Contact`:
```bash
...
ERROR in ./src/Contact/contactForm.ts 6:0-41
Module not found: Error: Can't resolve '@kor-ui/kor/components/textarea' in '/var/www/html/src/Contact'
 @ ./src/Contact/Contact.ts 3:0-23
 @ ./index.ts 4:0-48 9:49-56

webpack 5.11.1 compiled with 1 error and 1 warning in 6708 ms
...
```

**Comment by litneet64** [comment]
Looking on your previous commits on #50 and the webpack config, I found that you missed the line for textarea under the entry part in `webpack.dev.config.js`:

```js
...
    "components/tabs": './components/tabs/index.ts',
    "components/tag": './components/tag/index.ts',
    "components/text": './components/text/index.ts',
    "components/text": './components/text/index.ts',
    "components/toggle": './components/toggle/index.ts',
...
```

**Comment by eduferfer** [comment]
@litneet64 thanks for the catch and PR, I have just merged and published patch 1.5.1 👍🏽
